src/pages/Lending/hooks/useLendingQuoteQuery.ts

```diff
@@ -58,17 +59,26 @@ const selectLendingQuoteQuery = memoize(
     )
       .times(collateralAssetMarketData.price)
       .toString()
-    const userCurrencyToUsdRate = selectUserCurrencyToUsdRate(store.getState())
+
+    const quoteCollateralAmountFiatUsd = bn(quoteCollateralAmountFiatUserCurrency)
+      .times(bn(1).div(userCurrencyToUsdRate))
```

Comment:
```suggestion
      .div(userCurrencyToUsdRate)
```

🧠

Reply:
same for the others heh

Reply:
Might as well for cleanliness! ad3e79356b4c923b7c4c215792ec21f26a94b126